### DISCUSSION TITLE
Parallel Comparison

### DISCUSSION THREAD

**Discussion by Lencripe** [jplag/JPlag#818] [Q&A] [answered]
HI 
I'm having trouble running the parallel comparison strategy with v4.1.0 is this argument still supported with the new CLI ?

**Comment by tsaglam** [comment, marked as answer]
Starting from v4.0.0, the parallel comparison is enabled by default. The CLI parameter is thus removed. The reason for this is that we improved the parallel configuration to be even more efficient, thus being useful even for small datasets. For you, that just means you can run JPlag as is, and the matches will be calculated in parallel.
